tests/Generator/TwigGeneratorTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Ahc\Phint\Test;
+
+use Ahc\Phint\Generator\TwigGenerator;
+use PHPUnit\Framework\TestCase;
+
+class TwigGeneratorTest extends TestCase
+{
+    protected $templatePath = __DIR__ . '/../fixtures/twig';
+
+    public function setUp()
+    {
```

Comment:
should it be `setUpBeforeClass()`?

(and we can remove fixtures in `tearDownAfterClass()`)

Reply:
It's up to you. I think the class will be destructed in the end of the PHP process.